source/overloading.tex

```diff
@@ -1571,23 +1571,49 @@
 with the assumption that any dependent base class has
 no virtual functions and no virtual base classes, and
 the initializer is a non-empty \grammarterm{braced-init-list} or
-parenthesized \grammarterm{expression-list},
+parenthesized \grammarterm{expression-list}, and
+there are no \grammarterm{deduction-guide}{s} for \tcode{C},
 the set contains an additional function template,
 called the \defnadj{aggregate deduction}{candidate}, defined as follows.
 Let $x_1, \dotsc, x_n$ be the elements
 of the \grammarterm{initializer-list} or
 \grammarterm{designated-initializer-list}
 of the \grammarterm{braced-init-list}, or
 of the \grammarterm{expression-list}.
-For each $x_i$, let $e_i$ be the corresponding element
+For each $x_i$, let $e_i$ be the corresponding aggregate element
 of \tcode{C} or of one of its (possibly recursive) subaggregates
-that would be initialized by $x_i$\iref{dcl.init.aggr}
-if brace elision is not considered for any element
-that has a dependent type.
-If there is no such element $e_i$, the program is ill-formed.
+that would be initialized by $x_i$\iref{dcl.init.aggr} if
```

Comment:
The paper wants a ":" added after "if".

Reply:
Yes, but I don't want it. (This is a run-on sentence.)